src/localforage.js

```diff
@@ -118,50 +118,58 @@ class LocalForage {
         // If the options argument is an object, we use it to set values.
         // Otherwise, we return either a specified config value or all
         // config values.
-        if (typeof options === 'object') {
-            // If localforage is ready and fully initialized, we can't set
-            // any new configuration values. Instead, we return an error.
-            if (this._ready) {
-                return new Error(
-                    "Can't call config() after localforage " + 'has been used.'
-                );
-            }
-
-            for (let i in options) {
-                if (i === 'storeName') {
-                    options[i] = options[i].replace(/\W/g, '_');
+        try {
+            if (typeof options === 'object') {
+                // If localforage is ready and fully initialized, we can't set
+                // any new configuration values. Instead, we return an error.
+                if (this._ready) {
+                    return new Error(
+                        "Can't call config() after localforage " + 'has been used.'
+                    );
                 }
 
-                if (i === 'version' && typeof options[i] !== 'number') {
-                    return new Error('Database version must be a number.');
+                for (let i in options) {
+                    if (i === 'storeName' && typeof options[i] !== 'string') {
```

Comment:
Reading this code it seems like this `if` check will fail (calling the `else` block below) for any option key that isn't `storeName`. You'll want to nest the check for `(typeof options[i] !== 'string')` inside this `if` statement or this will break other things.